### DISCUSSION TITLE
Getting the Realized/Realized by Classes with py-capellambse

### DISCUSSION THREAD

**Discussion by khammoutene** [dbinfrago/py-capellambse#179] [Q&A] [answered]
Hi,

I have a Class defined in a data package at System Analysis level, e.g. ClassSA.

ClassSA is realized by a class in the logical architecture: e.g. ClassLA.

I have "printed" both classes with py-capellambse (using the print statement) but I cannot see in either class reference to the other one: e.g. "ClassLA realizes ClassSA" or "ClassSA is realized by ClassLA".

Am I missing something?

Thank you

**Comment by khammoutene** [comment]
Hello. Any idea how I could get that information via py-capellambse?

Actually, the "Realized Classes" works bottom-up so, in a class properties window, we should read that the current class realizes the classes listed in the "Realized Classes" field.

**Comment by ewuerger** [comment, marked as answer]
Hi,

thank you for another bug that you found and sorry for the late reply: I created an issue (#195) and solved the issue within PR #196. With these changes you can access the wanted attribute, the realization object and even the reverse (i.e. on a realized class there is `realized_by`).

Cheers.
